src/components/CategoryList/CategoryList.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+export default function CategoryList({onFilter}) {
+  return (
+    <StyledCategoryList>
+      <StyledCategoryListItem>
+        <StyledButton onClick={() => onFilter('Alle')}>Alle</StyledButton>
+      </StyledCategoryListItem>
+      <StyledCategoryListItem>
+        <StyledButton onClick={() => onFilter('Atemübung')}>Atem</StyledButton>
+      </StyledCategoryListItem>
+      <StyledCategoryListItem>
+        <StyledButton onClick={() => onFilter('Augenübung')}>Augen</StyledButton>
+      </StyledCategoryListItem>
+      <StyledCategoryListItem>
+        <StyledButton onClick={() => onFilter('Bewegung')}>Bewegung</StyledButton>
+      </StyledCategoryListItem>
```

Comment:
The next step would be to put your four filterstrings in an array and map them here with your `StyledCategoryListItem` and `StyledButton` component.